source/elements/oneMKL/source/domains/spblas/matrix_view.rst

```diff
@@ -0,0 +1,130 @@
+.. SPDX-FileCopyrightText: 2024 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_sparse_matrix_view_and_descr:
+
+Matrix view
+===========
+
+.. _onemkl_sparse_matrix_descriptor:
+
+matrix_descr
+------------
+
+.. rubric:: Definition
+
+.. code:: cpp
+
+   namespace oneapi::mkl::sparse {
+
+       enum class matrix_descr {
+           general,
+           symmetric,
+           hermitian,
+           triangular,
+           diagonal,
+       };
+
+   }
+
+.. container:: section
+
+   .. rubric:: Description
+
+   The matrix descriptor describes how an operation should interpret the data.
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 20 80
+
+      * - Value
+        - Description
+      * - ``general``
+        - General case, use complete data.
+      * - ``symmetric``
+        - View as symmetric, use given triangular part.
+      * - ``hermitian``
+        - View as hermitian, use given triangular part.
+      * - ``triangular``
+        - View as triangular, use given triangular part.
+      * - ``diagonal``
+        - View as diagonal, use only main diagonal values.
+
+.. _onemkl_sparse_matrix_view:
+
+matrix_view
+-----------
+
+.. rubric:: Definition
+
+.. code:: cpp
+
+   namespace oneapi::mkl::sparse {
+
+       struct matrix_view {
+           matrix_descr type_view = matrix_descr::general;
+           uplo uplo_view = uplo::lower;
+           diag diag_view = diag::nonunit;
+
+           matrix_view() = default;
+
+           matrix_view(matrix_descr type_view);
+       };
+
+   }
+
+.. container:: section
+
+   .. rubric:: Description
+
+   The matrix view holds information to specify which part of the matrix should
+   be read without changing the matrix's data.
+
+   See :ref:`onemkl_sparse_matrix_descriptor`, :ref:`onemkl_enum_uplo` and
+   :ref:`onemkl_enum_diag` fot a description of the members.
+
+   The ``uplo_view`` member is only used if ``type_view`` is ``symmetric``,
+   ``hermitian`` or ``triangular``.
```

Comment:
Let's be even more explicit here to match with `diag_view` description:
```suggestion
   The ``uplo_view`` member is only used if ``type_view`` is ``symmetric``,
   ``hermitian`` or ``triangular``. It is ignored if ``type_view`` is ``general``.
```

Reply:
I rephrased this in https://github.com/oneapi-src/oneAPI-spec/pull/522/commits/15139cc39ec308179725fd368f473f0d29d57ed2. It is also matching the description with `diag_view` and avoids listing all the values ``type_view`` can take.